crates/core_simd/src/swizzle.rs

```diff
@@ -364,4 +364,131 @@ where
 
         (Even::swizzle2(self, other), Odd::swizzle2(self, other))
     }
+
+    /// Takes a slice of a vector to produce a shorter vector.
+    ///
+    /// This is equivalent to computing `&self[OFFSET..OFFSET+LEN]` on
+    /// the underlying array.
+    ///
+    /// ```
+    /// # #![feature(portable_simd)]
+    /// # use core::simd::Simd;
```

Comment:
This needs to look more like this:
https://github.com/rust-lang/portable-simd/blob/8dcb4d5ef745dcbe62c77f25415c21fcbe9f165c/crates/core_simd/src/elements/int.rs#L18-L21